README.md

```diff
@@ -0,0 +1,30 @@
+# osd-network-verifier
+
+A cli and set of libraries that validates the preconfigured networking components for some osd options.
+
+## Overview
+
+osd-network-verifier can be used prior to the installation of osd/rosa clusters to ensure the pre-requirements are valid for various network options.
+
+### Workflow for egress
+
+* it creates an instance in the target vpc/subnet and wait till the instance gets ready
+* when the instance is ready, an `userdata` script is run against the instance to install appropriate packages, primarily docker
+* the validation is run against the vpc/subnet as containerized https://github.com/openshift/osd-network-verifier/tree/main/build 
```

Comment:
Probably note that this is done by the `userdata` script. The current description suggests the userdata script only installs packages, but it does both install packages and run the network-validator-test image.